llvm/test/Analysis/ScalarEvolution/pr120442.ll

```diff
@@ -0,0 +1,35 @@
+; RUN: opt < %s -disable-output "-passes=print<scalar-evolution>" -scalar-evolution-max-iterations=0  -scalar-evolution-classify-expressions=0  2>&1 | FileCheck %s
+
+declare void @llvm.assume(i1)
+
+define void @pr120442() {
```

Comment:
Can you add a brief comment explaining what the test checks?

Reply:
Thanks, added a comment